client/packages/requisitions/src/RequestRequisition/api/index.ts

```diff
@@ -1,2 +1,7 @@
 export * from './hooks';
 export * from './operations.generated';
+export {
+  RequestLineFragment,
+  ItemWithStatsFragment,
+  ItemInformationFragment,
+} from '@openmsupply-client/system';
```

Comment:
RequestLineFragment lived in higher level package, didn't want to introduce circular dependency, so moved the fragment generation to system and re-exported here